FUNDING.md

```diff
@@ -34,13 +35,9 @@ _List borrowed from [ESLint: "Funding ESLint's Future."](https://eslint.org/blog
 
 See all past expenses on our [Open Collective.](https://opencollective.com/astrodotbuild)
 
-### Eligibility
-
-**Employees, contractors, and consultants of The Astro Technology Company are not eligible to receive funds from Open Collective.** These funds exist solely to serve the larger Astro open-source community.
-
 ### Core Maintainer Stipend
 
-A monthly stipend of up to $1000 per calendar month is available to [eligible](#eligibility) core maintainers. Any time spent improving or supporting the project in some way counts towards this stipend: writing code, writing documentation, triaging issues, offering support in our Discord server, attending meetings (and meetups) on behalf of Astro, contributing to our upstream dependencies, and so on.
+A monthly stipend of up to $1000 per calendar month is available to all core maintainers. Any time spent improving or supporting the project in some way -- outside of an existing contract or agreement with The Astro Technology Company -- counts towards this stipend: writing code, writing documentation, triaging issues, offering support in our Discord server, attending meetings (and meetups) on behalf of Astro, contributing to our upstream dependencies, and so on.
```

Comment:
```suggestion
A stipend of up to $1000 per calendar month is available to all core maintainers. Any time spent improving or supporting the project in some way counts towards this stipend: writing code, writing documentation, triaging issues, offering support in our Discord server, attending meetings (and meetups) on behalf of Astro, contributing to our upstream dependencies, and so on. Any work that is covered by a separate contract or agreement with The Astro Technology Company or Open Collective does not count towards this stipend. For example, core maintainers under full time contracts with The Astro Technology Company cannot also claim a stipend, and any core maintainer contracted to perform specific work cannot also claim a stipend for that contracted work.
```

Reply:
I agree that's better wording. This also covers the case of someone that has a custom contract that isn't with TATC but Astro itself like me.